### PR TITLE
Polish on nivo horizontal graph 

### LINKED ISSUE #1019: Nivo bars text overlap on smaller charts
related #1013

Found that smaller charts in general have quite a fair bit of styling annoyances.
Minor (?) stuff mostly, maybe we won't bother fixing it(?)...

detailed spending - sobjs main offender I've taken notice to

image
- [x] x-axis labels are poorly scaled/overlapping

image
- [ ] Labels on the right aren't centered with the bars

image
- [ ] y-axis labels overlapping

### PR DESCRIPTION
**Pull request by IvanFeng1**
closes #1019

### CODE REVIEW

**Comment by IvanFeng1** [comment]
https://github.com/TBS-EACPD/infobase/blob/034fcc11c3c14630918f4d9c47c434d3c723f2e9/client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js#L314-L317

The values I used here are basically just values for the offset are values that make the labels align with the bar

File: client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js [hunk 1]
```diff
@@ -296,10 +296,11 @@ class DetailedProgramSplit extends React.Component {
 
     // Increase height of the graph region for y-axis labels to have sufficient room
     // This is required to corretly display the labels when too many programs are present
-    const divHeight = Math.max(
-      650 * (graph_ready_data.length / 30) * 2,
-      80 // "magic" minimum height for the graph to render as desired
+    let divHeight = Math.max(
```
Comment (ktw1016) [inline client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js:299]: I think `const minDivHeight` here

File: client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js [hunk 2]
```diff
@@ -296,10 +296,11 @@ class DetailedProgramSplit extends React.Component {
...
+      1000 * (graph_ready_data.length / 30) * 2,
+      100 // "magic" minimum height for the graph to render as desired
     );
+    divHeight = Math.min(divHeight, 1200); // max height of 1200 to prevent large graphs from becoming too large
```
Comment (ktw1016) [inline client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js:303]: and `const divHeight = Math.min(minDivHeight...)` is clearer
Or you could do something like
```
_.chain([...])
        .max()
        .thru((max_val) => [max_val, 1200])
        .min()
        .value()
```

**Review by ktw1016 — APPROVED** [review]
Other than the minor comment, looks good. Does this close the issue? Not overlapping anymore and labels on the right are centered on my screen

**Comment by IvanFeng1** [comment]
> Other than the minor comment, looks good. Does this close the issue? Not overlapping anymore and labels on the right are centered on my screen

I think this closes the issue.

**Comment by ktw1016** [comment]
Merge ahead and close the issue please. Thanks!

**Pull request merged**